src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="search">
+      <div class="search-bar search-bar--big" data-qa="big">
+        <div class="search-bar__icon-big" data-qa="keypress"></div>
+        <input
+          class="search-bar__input-big"
```

Comment:
```suggestion
      <div class="search-bar" data-qa="big">
        <div class="search-bar__icon" data-qa="keypress"></div>
        <input
          class="search-bar__input"
```

no need to mark elements or modifiers as big.  styles for big form  in classes without any modifiers and change necessary classes for small form in --small modifiers